packages/amplify-e2e-tests/src/__tests__/guardrail-test.ts

```diff
@@ -0,0 +1,96 @@
+import {
```

Comment:
Change the test file name to describe what's being tested and ensure it ends in `.test.ts`. Perhaps something like `parameter-store.test.ts`?